src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedWorkspaceSymbolFeature.java

```diff
@@ -24,6 +24,6 @@ public class UserDefinedWorkspaceSymbolFeature extends LSPWorkspaceSymbolFeature
     public boolean supportsGotoClass() {
         UserDefinedLanguageServerDefinition serverDefinition = (UserDefinedLanguageServerDefinition) getClientFeatures().getServerDefinition();
         ClientConfigurationSettings clientConfiguration = serverDefinition.getLanguageServerClientConfiguration();
-        return clientConfiguration != null ? clientConfiguration.workspaceSymbols.supportsGotoClass : super.supportsGotoClass();
+        return clientConfiguration != null ? clientConfiguration.workspaceSymbol.supportsGotoClass : super.supportsGotoClass();
```

Comment:
The JSON clientConfiguration.json uses "workspaceSymbol", it should mapped with the workspaceSymbol field.